src/SummaryTables.jl

```diff
@@ -40,6 +41,7 @@ include("infrastructure/cells.jl")
 include("infrastructure/special_cell_values.jl")
 include("infrastructure/tables.jl")
 
+include("table_functions/simple_table.jl")
 include("table_functions/table_one.jl")
 include("table_functions/listingtable.jl")
 include("table_functions/summarytable.jl")
```

Comment:
What's your thinking on the naming scheme here @jkrumbiegel? Two have `_`, and two don't.

Reply:
Agreed that `tableone` isn't as nice to read.

"simple table" naming sounds fine to me, and does signal that it won't have lots of customisation, which I think was what we wanted here.

I'd go with `simple_table` and then we also look towards deprecating the other non-`_` names in favour of consistent use of `_` to separate words in future?

Reply:
ok sure we can also go `_` all the way, it's not so "julian" but maybe more internally consistent then